tests/test_search_aggregates.py

```diff
@@ -143,16 +150,16 @@ def simple_main_summary(generate_main_summary_data):
 @pytest.fixture()
 def generate_exploded_data(define_dataframe_factory):
     return define_dataframe_factory(map(to_field, [
-        ('submission_date', '20170101', StringType(), False),
-        ('country',         'DE',       StringType(), True),
-        ('locale',          'de',       StringType(), True),
-        ('search_cohort',   None,       StringType(), True),
-        ('app_version',     '54.0.1',   StringType(), True),
-        ('distribution_id', None,       StringType(), True),
-        ('addon_version',   '0.9.5',    StringType(), False),
-        ('engine',          'google',   StringType(), False),
-        ('source',          'urlbar',   StringType(), False),
-        ('count',           4,          LongType(),   False),
+        ('submission_date', '20170101',    StringType(), False),
```

Comment:
I think I would prefer if we didn't pad the spacing in these structures as it makes diffs harder to read. Also, I don't use an editor that makes it easy to add the appropriate level of padding.

Reply:
I'm starting to agree with you. The upcoming search_clients_daily PR has some tests that break this padding. I'll remove the padding generally in that PR. SG?